static/sass/_snapcraft_market.scss

```diff
@@ -56,15 +56,19 @@
   // Stick the Revert, Apply buttons to the top
   .snapcraft-p-sticky {
     background: $color-x-light;
-    margin-bottom: $sp-small;
     position: sticky;
     top: 0;
     z-index: 1;
 
     & .row {
+      padding-bottom: $sp-medium;
       padding-top: $sp-medium;
     }
 
+    &.sticky-shadow {
+      box-shadow: 1px 1px 10px rgba(0, 0, 0, .2);
```

Comment:
That is the rationale for split from Vanillas tabbed component?

Reply:
This does not change the styling from the tabs, it changes a sticky element that we have on the listing page:

![image](https://user-images.githubusercontent.com/8167677/51987307-1822ad00-249a-11e9-929e-6f24d828a28d.png)

![image](https://user-images.githubusercontent.com/8167677/51987311-1ce76100-249a-11e9-894c-dec3864a31b5.png)

Right now when it floats it can feel like you are in the same part of the page without context